eva-events-db-schema/src/main/resources/com/razz/eva/events/db/V001__create_events.sql

```diff
@@ -46,38 +48,41 @@ LANGUAGE plpgsql;
 CREATE TRIGGER check_idempotency_key_on_update
     BEFORE UPDATE ON uow_events
     FOR EACH ROW
+    WHEN (NEW.idempotency_key IS NOT NULL)
```

Comment:
Now it doesn't trigger on inserts/updates of rows which have no `idempotency_key`.